Make command failure

OS: MacOS Big Sur 11.1

The command `make` fails with the following error:
```bash
cp -t bundle index.html spectral.yml spectral-full.yml spectral-security.yml spectral-generic.yml
cp: illegal option -- t
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvXc] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvXc] source_file ... target_directory
make: *** [copy_public] Error 64
```

--- a/Makefile
+++ b/Makefile
@@ -38,7 +38,8 @@ copy_public:
 	cp public/js/* bundle/js/
 	cp public/css/* bundle/css/
 	cp -r public/icons bundle/icons/
-	cp -t bundle index.html $(RULE_FILES)
+	cp index.html bundle/
+	cp ${RULE_FILES} bundle/
 
 # Merge all rules into spectral.yml
 rules: setup $(RULE_FILES)

